### PR TITLE
upgrade kymaVersion to 1.24.0 in resources/KYMA_VERSION

### PR DESCRIPTION
**Pull request by lauraanddola**
**Description**
try to implement the task [Upgrade Kyma on KCP to 1.23.1 -> 1.24.0](https://github.tools.sap/kyma/backlog/issues/1653)

### CODE REVIEW

**Review by koala7659 — DISMISSED** [review]
LGTM

**Comment by lauraanddola** [comment]
Hi dev, [pre-main-control-plane-gke-provisioner-integration](https://status.build.kyma-project.io/view/gs/kyma-prow-logs/pr-logs/pull/kyma-project_control-plane/877/pre-main-control-plane-gke-provisioner-integration/1427877183765876736) failed because of  `oct-tp-testsuite-all-kcp-provisioner-tests-0` image nnot found (details see  https://github.tools.sap/kyma/backlog/issues/1749 ) which is the known [issue 794 ](https://github.com/kyma-project/control-plane/issues/794).

File: components/provisioner/README.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ This table lists the environment variables, their descriptions, and default valu
 | Parameter | Description | Default value |
 |-----------|-------------|---------------|
 | **APP_ADDRESS** | Runtime Provisioner's address with the port | `127.0.0.1:3000` |
+| **APP_METRICS_ADDRESS** | Runtime Provisioner Metrics's address with the port | `127.0.0.1:9000` |
```
Comment (majakurcius) [inline components/provisioner/README.md:44]:
```suggestion
| **APP_METRICS_ADDRESS** | Runtime Provisioner Metrics' address with the port | `127.0.0.1:9000` |
```

**Review by majakurcius — APPROVED** [review]

**Review by koala7659 — APPROVED** [review]
LGTM

**Pull request merged**